manifests/client.pp

```diff
@@ -40,6 +40,7 @@
   Optional[Bacula::Yesno] $pki_encryption      = undef,
   Optional[String]        $pki_keypair         = undef,
   Optional[String]        $pki_master_key      = undef,
+  Optional[String]        $plugin_dir          = undef,
```

Comment:
Can you please add documentation for this parameter by adding a `@param` line in the beginning of the file?